VS_Project/Pokerapp/Pokerapp/spelers-uitschakelen.cshtml

```diff
@@ -0,0 +1,93 @@
+﻿@using WebMatrix.Data
+
+@{
+    string connectionString = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Database.mdf;Integrated Security=True";
+    string provider = "System.Data.SqlClient";
+    Database db = Database.OpenConnectionString(connectionString, provider);
+
+    //Insert Nick into Players
+    string name = "Nick";
+    int room_code = 1234567;
```

Comment:
Je voert hier static info in, zou je niet gewoon de speler pakken die je aanklikt?